static/app/views/releases/components/sideTable.tsx

```diff
@@ -0,0 +1,38 @@
+import styled from '@emotion/styled';
+
+import {space} from 'sentry/styles/space';
+
+type Props = {
+  children?: React.ReactNode;
+  type?: undefined | 'error' | 'warning';
+};
+
+export const Table = styled('dl')<{noMargin?: boolean}>`
+  ${p => (p.noMargin ? 'margin-bottom: 0;' : null)}
+`;
+
+export function TableRow({type, children}: Props) {
+  return <Row type={type}>{children}</Row>;
+}
```

Comment:
avoid exporting things called Table and TableRow. these components seem pretty specific to your use case

Reply:
good call
will update